io/zenoh-transport/src/multicast/link.rs

```diff
@@ -234,12 +234,13 @@ async fn tx_task(
                 let bytes = batch.as_bytes();
                 link.write_all(bytes).await?;
                 // Keep track of next SNs
-                if let Some(sn) = batch.sn.reliable {
-                    next_sns[priority].reliable = sn.next;
-                }
-                if let Some(sn) = batch.sn.best_effort {
-                    next_sns[priority].best_effort = sn.next;
-                }
+                // unimplemented!();
+                // if let Some(sn) = batch.sn.reliable {
+                //     next_sns[priority].reliable = sn.next;
+                // }
+                // if let Some(sn) = batch.sn.best_effort {
+                //     next_sns[priority].best_effort = sn.next;
+                // }
```

Comment:
some logic seems to disappear here. forgotten or no longer needed?

Reply:
Good catch. Fixed.